tests/run/t920.scala

```diff
@@ -1,20 +0,0 @@
-object Test {
```

Comment:
t920 fails on MacOS. See #1077. I'll push a version in pending.

On Mon, Feb 15, 2016 at 3:01 PM, Guillaume Martres <notifications@github.com
> wrote:

> In tests/run/t920.scala
> <https://github.com/lampepfl/dotty/pull/1082#discussion_r52901262>:
>
> > @@ -1,20 +0,0 @@
> > -object Test {
>
> Why was this test deleted?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/lampepfl/dotty/pull/1082/files#r52901262>.
>



-- 
Martin Odersky
EPFL